src/pages/home/ReportScreen.tsx

```diff
@@ -102,7 +102,7 @@ type ReportScreenProps = OnyxHOCProps & CurrentReportIDContextValue & ReportScre
 function getReportID(route: ReportScreenNavigationProps['route']): string {
     // The report ID is used in an onyx key. If it's an empty string, onyx will return
     // a collection instead of an individual report.
-    return String(route.params?.reportID || 0);
+    return String(route.params?.reportID || '');
```

Comment:
I believe this change has caused this composer issue

https://github.com/Expensify/App/assets/50919443/4aaaba59-9f82-4c8b-adbb-6ef6d4286e52

1. Open a chat
2. Go to settings
3. Go back to chat
4. It only happens once for each chat, so switch to another chat if you want to try it again

Returning an empty string will fetch the whole collection of onyx data as pointed out in the comment, e.g.,
`reportIsComposerFullSize_`, `report_`

_this was previously fixed in https://github.com/Expensify/App/pull/31602_

Reply:
Yeah, `getReportID` should return 0 as a fallback and have a separate variable that returns an empty string as a fallback, just like the old code.

Reply:
Hi guys, this PR introduced this deploy blocker https://github.com/Expensify/App/issues/39361, I will revert while we ensure that returning 0 here is the correct solution